website/static/js/profile.js

```diff
@@ -731,6 +731,14 @@ SocialViewModel.prototype.unserialize = function(data) {
     var self = this;
     var websiteValue = [];
     $.each(data || {}, function(key, value) {
+        if (key === 'scholar') {
```

Comment:
@alexschiller that change was done based on the example osf user I had shown him, but you are right. It should be done for all social profile links.